util/mcu_gen.py

```diff
@@ -319,24 +326,6 @@ def main():
                         default="",
                         help="CPU type (default value from cfg file)")
 
-    parser.add_argument("--bus",
```

Comment:
add them back pls, they should have higher priority

Reply:
fixed